mobile-widgets/qml/main.qml

```diff
@@ -334,7 +334,7 @@ Kirigami.ApplicationWindow {
 						name: ":/icons/cloud_sync.svg"
 					}
 					text: qsTr("Manual sync with cloud")
-					enabled: Backend.cloud_verification_status === Enums.CS_VERIFIED
+					visible: Backend.cloud_verification_status !== Enums.CS_NOCLOUD
```

Comment:
well - that's a behavioral change that might create confusion.
If the user has requested to create an account, but not verified their PIN, the entry will now be available, but the sync won't work.

Reply:
For what I can tell it's the other way round - in the old version the 'Manual synch' button is showing but cannot be clicked, making it appear 'dead' to the user. Now it is hidden, like the 'auto synch switch' button.